### PR TITLE
Revise Data Sharing Consent page UI

### PR DESCRIPTION
**Pull request by asadiqbal08**
@douglashall @brittneyexline FYI. 

**Description:** 
With the change to the main (track selection) Landing page, we will need to match the DSC experience.

Acceptance Criteria

1. Add Enterprise message to DSC page
2. Static text
3. Enterprise logo parameter
4. Enterprise name parameter
5. Site name parameter

**JIRA:** [ENT-352](https://openedx.atlassian.net/browse/ENT-352)

**Dependencies:** N/A

**Merge deadline:** 2-June-2017

**Testing instructions:**

1. visit https://ec-352.sandbox.edx.org/login
2. Click on `testshib_1` button
3. login with testshib credentials e.g. myself/myself
4. You can see the data sharing consent page with new design.

**Current UI Screen-shot**: 

screen shot 2017-05-25 at 6 12 51 pm

**Merge checklist:**

- [ ] All reviewers approved
- [x] CI build is green
- [x] Version bumped
- [x] Change log record added
- [x] Commits are (reasonably) squashed
- [x] PR author is listed in AUTHORS

**Post merge:**
- [ ] Create a tag
- [ ] Delete working branch (if not needed anymore)

#### NOTE: I will build a sandbox for UI testing.

### CODE REVIEW

File: enterprise/templates/enterprise/grant_data_sharing_permissions.html [hunk 1]
```diff
@@ -41,8 +41,20 @@
...
+                    <img class="enterprise-logo" src="${ enterprise_customer.branding_configuration.logo.url }" alt="${ enterprise_customer.name }"/>
+                % endif
+                <div class="partnered-text">
+                    ${ enterprise_welcome_text }
+                </div>
+            </div>
+            <div class="col-7 border-left">
+                <main>
+                    <div class="consent-container">
         <h2 class="consent-title">${ consent_message_header }</h2>
```
Comment (douglashall) [inline enterprise/templates/enterprise/grant_data_sharing_permissions.html]: Please fix the indentation here.

File: enterprise/templates/enterprise/grant_data_sharing_permissions.html [hunk 2]
```diff
@@ -98,7 +109,7 @@ <h2 class="consent-title">${ consent_message_header }</h2>
           </p>
         </div>
       </div>
-      <div id="consent-confirmation-modal" class="modal">
+                    <div id="consent-confirmation-modal" class="modal">
```
Comment (douglashall) [inline enterprise/templates/enterprise/grant_data_sharing_permissions.html]: Please fix the indentation here.

**Comment by asadiqbal08** [comment]
@saleem-latif @zubair-arbi can you guys please review it ?

**Comment by zubair-arbi** [comment]
@asadiqbal08 you need to fix the alignment on `Safari` browser:
screen shot 2017-05-30 at 12 53 17 pm

**Comment by zubair-arbi** [comment]
@asadiqbal08 Please hide the enterprise logo on mobile view for consistency with enterprise course enrollment page and enterprise logistration pages.

**Comment by douglashall** [comment]
@asadiqbal08 I'm not seeing the DSC page when signing in as directed in the testing instructions. Is there something I am missing?

**Comment by asadiqbal08** [comment]
@douglashall try with some other user, may be sso-linked-user is allowed the data sharing consent. Let me know If you find any difficulties.

**Comment by douglashall** [comment]
@asadiqbal08 Yes, that worked. I used alterego/alterego on Testshib. Is there a way to reset the DSC for a given user?

**Comment by asadiqbal08** [comment]
Not sure, may be we need to use Django admin to reset the DSC for a user. However, for the quick UI display, you can register a new user in system.

**Comment by douglashall** [comment]
@brittneyexline Is there a way to reset DSC for a specific user in Django admin?

**Pull request merged**